lib/kitchen/directions/bake_footnotes/v1.rb

```diff
@@ -10,32 +10,12 @@ def bake(book:)
       book.body.element_children.only(Kitchen::PageElement,
                                       Kitchen::CompositePageElement,
                                       Kitchen::CompositeChapterElement).each do |page|
```

Comment:
just checking that composite chapters are meant to be included? since the variable name is `page`